client/src/context/useAuthContext.tsx

```diff
@@ -25,10 +29,19 @@ export const AuthProvider: FunctionComponent = ({ children }): JSX.Element => {
   const updateLoginContext = useCallback(
     (data: AuthApiDataSuccess) => {
       setLoggedInUser(data.user);
-      history.push('/dashboard');
+      history.push('/edit-profile');
```

Comment:
Won't this redirect every user who logs in regardless of whether they have a profile or not? :)